apps/interpreter/src/run-action.ts

```diff
@@ -23,6 +26,11 @@ export async function runAction(
       services,
       loggerFactory.createLogger(),
     );
+  if (options.parseOnly === true) {
+    const { model, services } = await parseModel(extractAstNodeFn, options);
+    const exitCode = model && services ? 0 : 1;
```

Comment:
At least in `parseModel` `model` and `services` are compared with `null`, not just falsy. Just for consistency I think checking for `null` here as well would make sense.